libgnucash/core-utils/gnc-filepath-utils.h

```diff
@@ -172,4 +172,5 @@ gchar *gnc_filepath_locate_ui_file (const gchar *name);
  */
 gchar *gnc_filepath_locate_doc_file (const gchar *name);
 
+gchar *gnc_log_all_paths (gboolean use_markup);
```

Comment:
This function should be in app-utils/gnc-help-utils. It also needs Doxygen documentation.